less/ff.less

```diff
@@ -0,0 +1,34 @@
+// Firefox-specific CSS rules to work around
+// invalid assumptions made in Bootstrap 3
+// (but fixed in Bootstrap 4)
+
+@media screen and (min--moz-device-pixel-ratio: 0) {
+  input[type=date].form-control,
```

Comment:
Could use the power of LESS to make this source a lot more concise but the output CSS wouldn't be much different and we're unlikely to be modifying this frequently/ever again so fine as is